src/main/java/org/opensearch/knn/index/memory/NativeMemoryCacheManagerDto.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.memory;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
```

Comment:
[Minor]
if you use @Value and @Builder then you don't need add @Getter and you can remove final from every variable as @Value will make class variables final.

Reply:
Makes sense. Will add.